README.example.md

```diff
@@ -95,16 +148,6 @@ If you need to make local changes to the env files, create a `.env.ENVIRONMENT.l
 
```

Comment:
maybe add a comment about just plain .env.local

Reply:
Believe it or not; I actually changed this line entirely, but then realised it was out of scope for this PR and reverted it :grinning:

Reply:
I think I'll just make that change directly in the main branch.

Reply:
I've updated this in main